packages/libs/pactjs-generator/src/index.ts

```diff
@@ -18,6 +18,11 @@ export * from './contract/generation/IContractDefinition';
  */
 export * from './contract/generation/generator';
 
+/**
+ * @alpha
+ */
+export * from './contract/generation/generatorV2';
+
```

Comment:
I'm not sure if it's needed to put the semver annotation here, if they're already at the function itself.